l10n_it_dichiarazione_intento/models/dichiarazione_intento.py

```diff
@@ -0,0 +1,203 @@
+
+    @api.multi
+    def _compute_used_amount(self):
+        for record in self:
+            dichiarazioni = self.env['dichiarazione.intento'].search([
+                ('date_start', '>=', '%s-01-01' % record.year),
+                ('date_end', '<=', '%s-12-31' % record.year),
+                ('type', '=', 'out'), ])
+            record.used_amount = sum([d.limit_amount for d in dichiarazioni])
+
+
+class DichiarazioneIntento(models.Model):
+
+    _name = 'dichiarazione.intento'
+    _description = 'Dichiarazione Intento'
+    _order = 'date_start desc,date_end desc'
+    _rec_name = 'display_name'
+
+    @api.model
+    def _default_currency(self):
+        return self.env.user.company_id.currency_id
+
+    display_name = fields.Char(
+        compute='_compute_clean_display_name', store=True)
+    number = fields.Char(copy=False)
+    date = fields.Date(required=True)
+    date_start = fields.Date(required=True)
+    date_end = fields.Date(required=True)
+    type = fields.Selection([('in', 'Suppliers'), ('out', 'Customers')],
+                            required=True, default='in')
+    partner_id = fields.Many2one('res.partner', string='Partner',
+                                 required=True)
+    partner_document_number = fields.Char(
+        required=True, string='Document Number',
+        help='Number of partner\'s document')
+    partner_document_date = fields.Date(
+        required=True, string='Document Date',
+        help='Date of partner\'s document')
+    taxes_ids = fields.Many2many('account.tax', string='Taxes',
+                                 required=True)
+    used_amount = fields.Monetary(compute='_compute_amounts', store=True)
+    limit_amount = fields.Monetary(required=True)
+    available_amount = fields.Monetary(compute='_compute_amounts', store=True)
+    currency_id = fields.Many2one('res.currency', string='Currency',
+                                  default=_default_currency,)
+    fiscal_position_id = fields.Many2one(
+        'account.fiscal.position', string='Fiscal Position', required=True,
+        domain=[('valid_for_dichiarazione_intento', '=', True)])
+    state = fields.Selection(
+        [('valid', 'Valid'), ('close', 'Close')],
+        compute='_compute_state', store=True)
+    line_ids = fields.One2many('dichiarazione.intento.line',
+                               'dichiarazione_id', string='Lines')
+
+    @api.model
+    def create(self, values):
+        # ----- Check if yearly platfond is enough
+        #       to create an in dichiarazione
+        if values.get('type', False) == 'in':
+            year = datetime.strptime(
+                values['date_start'], '%Y-%m-%d').strftime('%Y')
+            platfond = self.env.user.company_id.\
+                dichiarazione_yearly_limit_ids.filtered(
+                    lambda r: r.year == year)
+            if not platfond:
+                raise UserError(
+                    _('Define a yearly platfond for out documents'))
```

Comment:
Nella file `it.po` non compaiono temini tradotti relativi ad `out`, è un refuso della traduzione o il termine `out` è stato aggiunto per errore?